SteamKit2/Tests/NetHelpersFacts.cs

```diff
@@ -11,12 +11,12 @@ public class NetHelpersFacts
         public void GetMsgIPAddress()
         {
             Assert.Equal( 2130706433u, NetHelpers.GetMsgIPAddress( IPAddress.Loopback ).v4 );
-            Assert.Equal( new byte[] {
+            Assert.Equal( [
```

Comment:
... the target-typing picks this up as a byte array literal?

Reply:
Yes.